contracts/eth/nearbridge/contracts/AdminControlled.sol

```diff
@@ -0,0 +1,38 @@
+pragma solidity ^0.6;
+
+contract AdminControlled {
+    address public admin;
+    uint public paused;
+
```

Comment:
Add the constructor there, please. Thus the approach of setting admin in inherited classes will be smoother and we can also set the initial flags (which functions are paused on the initialization) in the ctor.

Reply:
Please add to the constructor the ability to set flags